DigitalOffice/Tests/TimeManagementServiceTests/Commands/CreateWorkTimeCommandTests.cs

```diff
@@ -0,0 +1,43 @@
+using FluentValidation;
+using Moq;
+using NUnit.Framework;
+using TimeManagementService.Commands;
+using TimeManagementService.Commands.Interfaces;
+using TimeManagementService.Database.Entities;
+using TimeManagementService.Mappers.Interfaces;
+using TimeManagementService.Models;
+using TimeManagementService.Repositories.Interfaces;
+
+namespace TimeManagementServiceUnitTests.Commands
+{
+    public class CreateWorkTimeCommandTests
+    {
+        private Mock<IValidator<CreateWorkTimeRequest>> validatorMock;
+        private Mock<IMapper<CreateWorkTimeRequest, DbWorkTime>> mapperMock;
+        private Mock<IWorkTimeRepository> repositoryMock;
+
+        private ICreateWorkTimeCommand command;
+
+        [SetUp]
+        public void SetUp()
+        {
+            validatorMock = new Mock<IValidator<CreateWorkTimeRequest>>();
+            mapperMock = new Mock<IMapper<CreateWorkTimeRequest, DbWorkTime>>();
+            repositoryMock = new Mock<IWorkTimeRepository>();
+
+            command = new CreateWorkTimeCommand(validatorMock.Object, mapperMock.Object, repositoryMock.Object);
+        }
+
+        [Test]
+        public void ATest()
+        {
+            Assert.Pass();
+        }
+
+        [Test]
+        public void BTest()
+        {
+            Assert.Pass();
+        }
```

Comment:
I can not understand what it is.

Reply:
In a comment to the pull-request, I wrote that I have not done it yet.

Reply:
Fixed.